op-e2e/faultproofs/challenge_preimage_test.go

```diff
@@ -0,0 +1,69 @@
+package faultproofs
+
+import (
+	"context"
+	"testing"
+
+	"github.com/ethereum-optimism/asterisc/op-e2e/e2eutils/disputegame"
+	op_e2e "github.com/ethereum-optimism/optimism/op-e2e"
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/challenger"
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/disputegame/preimage"
+	op_e2e_faultproofs "github.com/ethereum-optimism/optimism/op-e2e/faultproofs"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+)
+
+func TestChallengeLargePreimages_ChallengeFirst(t *testing.T) {
+	op_e2e.InitParallel(t)
+	ctx := context.Background()
+	sys, _ := op_e2e_faultproofs.StartFaultDisputeSystem(t)
+	t.Cleanup(sys.Close)
+
+	disputeGameFactory := disputegame.NewAsteriscFactoryHelper(t, ctx, sys)
+	disputeGameFactory.StartChallenger(ctx, "Challenger",
+		challenger.WithAlphabet(sys.RollupEndpoint("sequencer")),
+		challenger.WithPrivKey(sys.Cfg.Secrets.Alice))
```

Comment:
Do we need this file in the Asterisc repo? it seems not to use Asterisc VM

Reply:
When we compare the implementation of `PreimageHelper` Method, 
Cannon:
```go
func (h *FactoryHelper) PreimageHelper(ctx context.Context) *preimage.Helper {
	opts := &bind.CallOpts{Context: ctx}
	gameAddr, err := h.factory.GameImpls(opts, cannonGameType)
	h.require.NoError(err)
	game, err := bindings.NewFaultDisputeGameCaller(gameAddr, h.client)
	h.require.NoError(err)
	vmAddr, err := game.Vm(opts)
	h.require.NoError(err)
	vm, err := bindings.NewMIPSCaller(vmAddr, h.client)
	h.require.NoError(err)
	oracleAddr, err := vm.Oracle(opts)
	h.require.NoError(err)
	return preimage.NewHelper(h.t, h.opts, h.client, oracleAddr)
}
```
Asterisc:
```go
func (h *AsteriscFactoryHelper) PreimageHelper(ctx context.Context) *preimage.Helper {
	opts := &bind.CallOpts{Context: ctx}
	gameAddr, err := h.Factory.GameImpls(opts, asteriscGameType)
	h.Require.NoError(err)
	game, err := op_bindings.NewFaultDisputeGameCaller(gameAddr, h.Client)
	h.Require.NoError(err)
	vmAddr, err := game.Vm(opts)
	h.Require.NoError(err)
	vm, err := bindings.NewRISCVCaller(vmAddr, h.Client)
	h.Require.NoError(err)
	oracleAddr, err := vm.Oracle(opts)
	h.Require.NoError(err)
	return preimage.NewHelper(h.T, h.Opts, h.Client, oracleAddr)
}
```
It still touches asterisc specific code, so I added those tests.